### ISSUE TITLE
SEO: Add meta tags

### ISSUE THREAD

**Issue by afterburn** [saving-satoshi/saving-satoshi#228]
As @benalleng pointed out in [this comment](https://github.com/saving-satoshi/saving-satoshi/pull/199#issuecomment-1454088002) we are missing meta tags that will help improve our listing in search engines.

We should make sure that on every page we include the proper [meta tags](https://ahrefs.com/blog/seo-meta-tags/) + [og meta tags](https://ahrefs.com/blog/open-graph-meta-tags/). Perhaps @ecurrencyhodler / @satsie / @GBKS can start thinking about what would be the proper content for the tags.

**Comment by ecurrencyhodler** [comment]
Thanks kevin for creating this issue. Fun fact, I actually have a little bit of a background in SEO. 

But let's hold push all SEO-related tasks to later.

**Comment by afterburn** [comment]
> Thanks kevin for creating this issue. Fun fact, I actually have a little bit of a background in SEO. 
> 
> But let's hold push all SEO-related tasks to later.

Ah that's great! Yeah, I just created these as a reminder for ourselves, these aren't needed for the demo or anything. 😁

**Comment by ecurrencyhodler** [comment]
Here's the snippet to add to the website.
```
///MetaData
<title>Saving Satoshi - Code your way through the mysteries of Bitcoin!</title>
  <meta name="description" content="A fun learning resource for developers to learn about how to build on bitcoin.">
  <meta property="og:type" content="website">
  <meta property="og:image" content="https://imagedelivery.net/wyrwp3c-j0gDDUWgnE7lig/6818cd1b-ea0d-49a4-ea14-d34c1bbae100/public">
  <meta property="og:title" content="Saving Satoshi - Code your way through the mysteries of Bitcoin!">
  <meta property="og:description" content="A fun learning resource for developers to learn about how to build on bitcoin.">
  <meta property="og:type" content="website">
  <meta property="og:url" content="https://savingsatoshi.com/">

///Schema
<script type="application/ld+json" class="schemantra">{
  "@context": "https://schema.org",
  "@type": "CreativeWork",
  "@id": "CreativeWork",
  "abstract": "Saving Satoshi is a light-hearted first point of contact for developers who are curious about bitcoin development. Saving Satoshi has a mix of technical text and code-based challenges, but the challenges should be doable for anyone with basic coding skills. The project is free and open source and all the art has been generated using text-to-image (aka AI) tools like Midjourney.",
  "creativeWorkStatus": "Published",
  "dateCreated": "2022-10-06",
  "sameAs": [
      "https://twitter.com/savingsatoshi",
      "https://makers.bolt.fun/project/saving-satoshi",
      "https://github.com/saving-satoshi"
    ],
  "datePublished": "2023-03-24",
  "description": "A fun learning resource for developers to learn about how to build on bitcoin.",
  "educationalLevel": "beginner",
  "genre": "Education Platform",
  "headline": "Code your way through the mysteries of bitcoin!",
  "isAccessibleForFree": true,
  "isFamilyFriendly": true,
  "keywords": "bitcoin, bitcoin developer, bitcoin development, btc",
  "learningResourceType": "Coding game platform",
  "name": "Saving Satoshi",
  "teaches": "Bitcoin, coding",
  "thumbnailUrl": "https://imagedelivery.net/wyrwp3c-j0gDDUWgnE7lig/6818cd1b-ea0d-49a4-ea14-d34c1bbae100/public"
}</script>
```

**Comment by Sash002** [comment]
Can we add more schema markup to the above code snippet like  'AggregateRating'  to provide user ratings and reviews or maybe 'VideoObject' could be used to indicate the presence of video content on the webpage?

**Comment by ecurrencyhodler** [comment]
We don't have user ratings and we don't have any video content.

**Comment by ecurrencyhodler** [comment]
@sabakhilji How do you feel about taking this issue on?

**Comment by benalleng** [comment]
Next13 makes the metatag layout substantially different from our current format, while it is relatively backwards compatible for the basic set for more advanced things we need to think about making a transition.

There are a lot of links in documentation to go through, I will likely just start pasting them here for my own sake and allow other people to look through them

https://nextjs.org/blog/next-13-2#turbopack-improvements

https://nextjs.org/blog/next-13-2

https://beta.nextjs.org/docs/upgrade-guide#step-3-migrating-nexthead

https://beta.nextjs.org/docs/api-reference/metadata

**Comment by ecurrencyhodler** [comment]
Sending this to backlog for now. Not necessary for v1.

**Comment by sabakhilji** [comment]
@ecurrencyhodler Can I have a look at it?

**Comment by ecurrencyhodler** [comment]
Sure! Sync with ben after tuesday on this issue and get back to me on it after you two scope it out a bit.

**Comment by afterburn** [comment]
Sorry I was working on missing document.titles and fixed this in #411

**Comment by benalleng** [comment]
It seems like the metadata for the live site isn't quite right.

![image](https://github.com/saving-satoshi/saving-satoshi/assets/108441023/a843b1e8-fa39-4331-b536-81e06d7d85eb)

When googling saving satoshi in chrome the website does not show up on the first page
